go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/foxcpp/go-mockdns v1.0.0
 	github.com/goccy/go-yaml v1.9.6
 	github.com/google/go-cmp v0.5.9
+	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
```

Comment:
We have been using go-shellwords: https://github.com/lima-vm/lima/blob/020d7ee10b219822d4c59d72c3764f3e1ee3baa6/pkg/qemu/qemu.go#L616

Not sure which one of them is better though

Reply:
Thanks for the pointer, I didn't even see this. 

Done in [ce109f95..648c47d4](https://github.com/lima-vm/lima/compare/ce109f9548a3ce3381a49eed03011c112d953e5d..648c47d49f8e81ebb738530cd593aa16401d90a6).